src/test/resources/mapping/expectedTransformationFields.json

```diff
@@ -1,1163 +1,1147 @@
-       "path": "$.item.electronicAccess[?(@.relationshipId=='5bfe1b7b-f151-4501-8cfa-23b321d5cd1e')].uri",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.electronic.access.uri.resource",
-       "displayNameKey": "item.electronic.access.uri",
-       "referenceDataValue": "Resource",
-       "path": "$.item.electronicAccess[?(@.relationshipId=='f5d0068e-6272-458e-8a81-b85e7b9a14aa')].uri",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.electronic.access.uri.version.of.resource",
-       "displayNameKey": "item.electronic.access.uri",
-       "referenceDataValue": "Version of resource",
-       "path": "$.item.electronicAccess[?(@.relationshipId=='3b430592-2e09-4b48-9a0c-0636d66b9fb3')].uri",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.enumeration",
-       "displayNameKey": "item.enumeration",
-       "path": "$.items[*].enumeration",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.holdingsrecordid",
-       "displayNameKey": "item.holdingsRecordId",
-       "path": "$.items[*].holdingsRecordId",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.hrid",
-       "displayNameKey": "item.hrid",
-       "path": "$.item.hrid",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.id",
-       "displayNameKey": "item.id",
-       "path": "$.item.id",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.itemnotetypeid.action.note",
-       "displayNameKey": "item.itemNoteTypeId",
-       "referenceDataValue": "Action note",
-       "path": "$.item[*].notes[?(@.itemNoteTypeId=='0e40884c-3523-4c6d-8187-d578e3d2794e' && (!(@.staffOnly) || @.staffOnly == false))].note",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.itemnotetypeid.binding",
-       "displayNameKey": "item.itemNoteTypeId",
-       "referenceDataValue": "Binding",
-       "path": "$.item[*].notes[?(@.itemNoteTypeId=='87c450be-2033-41fb-80ba-dd2409883681' && (!(@.staffOnly) || @.staffOnly == false))].note",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.itemnotetypeid.staffonly.action.note",
-       "displayNameKey": "item.itemNoteTypeId.staffOnly",
-       "referenceDataValue": "Action note",
-       "path": "$.item[*].notes[?(@.itemNoteTypeId=='0e40884c-3523-4c6d-8187-d578e3d2794e' && ((@.staffOnly) && @.staffOnly == true))].note",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.itemnotetypeid.staffonly.binding",
-       "displayNameKey": "item.itemNoteTypeId.staffOnly",
-       "referenceDataValue": "Binding",
-       "path": "$.item[*].notes[?(@.itemNoteTypeId=='87c450be-2033-41fb-80ba-dd2409883681' && ((@.staffOnly) && @.staffOnly == true))].note",
-       "recordType": "ITEM"
-     },
-     {
-       "fieldId": "item.materialtypeid.book",
-       "displayNameKey": "item.materialTypeId",
-       "referenceDataValue": "book",
```

Comment:
The removed entity for this contains the `referenceDataValue` and the new one does not. Bacuase of this translations on UI will look.

UI pattern for it now looks like `"item.materialTypeId": "Item - Material type - {value}",` where value is `referenceDataValue`.